json_encoder.go

```diff
@@ -121,7 +26,7 @@ type jsonEncoder struct {
 // NewJSONEncoder creates a JSON encoder, populating a minimal
 // set of Elastic common schema (ECS) fields.
 func NewJSONEncoder(cfg EncoderConfig) zapcore.Encoder {
```

Comment:
As discussed out of band, I think it makes sense to remove ecszap.NewJSONEncoder now.

Reply:
Removed in this PR now